test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2782,14 +2782,32 @@ def forward(self, x):
         self.run_test(Zero_(), x)
 
     @skipIfUnsupportedMinOpsetVersion(9)
-    def test_new_zero(self):
+    def test_new_zeros(self):
         class Zero_(torch.nn.Module):
             def forward(self, x):
-                return x.new_zeros(x.shape[2:])
+                return x.new_zeros(x.shape[1:2]), x.new_zeros(x.shape[2:], dtype=torch.long)
```

Comment:
curious, how was `new_zeros` exported before this?

Reply:
new_zeros was exporting okay. I just added this test to cover the case where dtype is different.